### ISSUE TITLE
How to use the illegality test function？

### ISSUE THREAD

**Issue by pyImagination** [kmaehashi/pypict#7]
Dear author, I have a question here. I don't know how to set parameters to support negative testing. In PICT, you only need to mark a '~' sign before the parameter to generate a negative case.How can i generatea negative cases  in pypict? I hope you can help me,thank you very  much!

**Comment by kmaehashi** [comment]
Unfortunately that is a feature of PICT Command; it is not available in PICT C API layer.
If you are familiar with PICT's model syntax, you can use `pypict.cmd.from_model(your_model_text)` which uses PICT Command, instead of `pypict.tools.from_dict(your_model_dict)` which uses PICT C API.

**Comment by pyImagination** [comment]
Yes,I got it. I had used to solve my problem by `pypict.cmd.from_model(your_model_text)` .Thank you very much!
